common/notification/actions.ts

```diff
@@ -412,6 +412,31 @@ const _notificationActions = {
             appointment: sampleAppointment,
         },
     },
+    missed_one_on_one_chat_message: {
+        description: 'Missed message in 1:1 chat',
+        sampleContext: {
+            sender: sampleUser,
+            recipient: sampleUser,
+            conversationId: sampleAppointment,
+            message: '',
+            totalUnread: '',
+            matchId: '',
+            loginToken: '',
+        },
+    },
+    missed_course_chat_message: {
+        description: 'Missed message in group chat',
+        sampleContext: {
+            sender: sampleUser,
+            recipient: sampleUser,
+            conversationId: sampleAppointment,
+            message: '',
+            totalUnread: '',
+            courseId: '',
+            courseName: '',
+            loginToken: '',
```

Comment:
This does not need to be in the sampleContext for this action, this is implicitly passed in (like the user)